### PR TITLE
Add a skip-prepare parameter for MaxThroughput

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes are proposed in this pull request?

We'll do a same maxThroughput test many time, so we don't want to change basedir or create&delete a lots of file before test.

### Why are the changes needed?

Add a parameter to supply a way to skip prepare step.

### Does this PR introduce any user facing changes?

Add a new parameter `--skip-prepare`

### CODE REVIEW

File: stress/shell/src/main/java/alluxio/stress/cli/suite/MaxThroughput.java [hunk 1]
```diff
@@ -73,7 +73,9 @@ public MaxThroughputSummary runSuite(String[] args) throws Exception {
 
     List<String> baseArgs = new ArrayList<>(Arrays.asList(args));
 
-    prepareBeforeAllTests(baseArgs);
+    if (mParameters.mSkipPrepare) {
```
Comment (yuzhu) [inline stress/shell/src/main/java/alluxio/stress/cli/suite/MaxThroughput.java:76]: i think you meant `!mParameters.mSkipPrepare` here

Comment (maobaolong) [reply stress/shell/src/main/java/alluxio/stress/cli/suite/MaxThroughput.java]: Exactly, Thanks for remind me

**Comment by maobaolong** [comment]
@yuzhu Thanks for remind me, I miss a `!` here, I fix  it by the latest commit, please take a another look.

**Review by yuzhu — APPROVED** [review]

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**